src/main/resources/site/parts/highmap/Highmap.jsx

```diff
@@ -167,13 +152,22 @@ function Highmap(props) {
   }
 
   return (
-    <section className='part-highmap container'>
+    <section className='xp-part part-highchart'>
```

Comment:
Er det litt forvirrende at det ser ut som parten her er `part-highchart`? Burde vi heller lage en felles klasse som tok for seg standard styling?